styles.css

```diff
@@ -41,7 +41,76 @@
     color: black;
 }
 
-.nav ul li:last-of-type {
-    border-right: none;
+footer {
+    align-items: center;
+    background-color: lightgray;
+    border-color: white;
+    border-style: solid;
```

Comment:
These border properties are adding 1px of white space in the footer borders, remove `border-color` and `border-style` properties from this footer
![image](https://user-images.githubusercontent.com/30848622/139608488-eebc131d-c3b5-4276-94b3-947be0b3a98e.png)